cumulus/parachains/runtimes/assets/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -38,8 +38,8 @@ use xcm_builder::{
 	EnsureXcmOrigin, FungiblesAdapter, HashedDescription, IsConcrete, LocalMint, NativeAsset,
 	NoChecking, ParentAsSuperuser, ParentIsPreset, RelayChainAsNative, SiblingParachainAsNative,
 	SiblingParachainConvertsVia, SignedAccountId32AsNative, SignedToAccountId32,
-	SovereignSignedViaLocation, TakeWeightCredit, TrailingSetTopicAsId, UsingComponents,
-	WeightInfoBounds, WithComputedOrigin, WithUniqueTopic,
+	SovereignSignedViaLocation, StartsWith, TakeWeightCredit, TrailingSetTopicAsId,
```

Comment:
Where did this move of `StartsWith` happen? Will need to be ported to the Fellowship repo. Perhaps remove Kusama and Polkadot changes from this PR?

Reply:
I could do a hack with re-exporting `StartsWith` from asset-common, so there wont be any changes to AssetHubKusama/AssetHubPolkadot here, but I am not sure if this is really needed. Maybe the only pros would be no need for `polkadot/locks-review` approve on this PR :)

It is just changed import, yes, I will do a follow-up PR to `polkadot-fellows`, but we need to merge this first to master here and wait probably for `1.3.0` crates.io release.